apps/bfd-pipeline/bfd-pipeline-app/src/main/assembly/bin/bfd-pipeline-app.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+scriptDirectory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+bfdPipelineAppJar="${scriptDirectory}/$(ls ${scriptDirectory} | grep '^bfd-pipeline-app-.*jar')"
+classpath="${bfdPipelineAppJar}:${scriptDirectory}/lib/*"
+mainClass="gov.cms.bfd.pipeline.app.PipelineApplication"
+[ -n "${BFD_JAVA_HOME}" ] && javaExecutable=${BFD_JAVA_HOME}/bin/java || javaExecutable=java
+
+exec ${javaExecutable} -cp ${classpath} $* ${mainClass}
```

Comment:
Not sure what this is doing, but good practice to quote vars `"${classpath}"` for example. What is the `$*` doing?

Reply:
`$*` is all arguments passed to the script.  I usually prefer `"$@"` though since it takes care of any arguments that contain spaces.  Putting quotes around `$*` puts all of them into a single string but `"$@"` inserts every argument individually as a quoted value.